fenix/app/src/main/res/values/strings.xml

```diff
@@ -1644,27 +1652,46 @@
     <string name="preferences_android_autofill_description">Fill usernames and passwords in other apps on your device.</string>
     <!-- Preference option for adding a login -->
     <string name="preferences_logins_add_login">Add login</string>
+    <!-- Preference option for adding a login -->
+    <string name="preferences_logins_add_login_2" tools:ignore="UnusedResources">Add password</string>
 
     <!-- Preference for syncing saved logins in Fenix -->
     <string name="preferences_passwords_sync_logins">Sync logins</string>
+    <!-- Preference for syncing saved logins in Fenix -->
+    <string name="preferences_passwords_sync_logins_2" tools:ignore="UnusedResources">Sync passwords</string>
     <!-- Preference for syncing saved logins in Fenix, when not signed in-->
     <string name="preferences_passwords_sync_logins_across_devices">Sync logins across devices</string>
+    <!-- Preference for syncing saved logins in Fenix, when not signed in-->
+    <string name="preferences_passwords_sync_logins_across_devices_2" tools:ignore="UnusedResources">Sync passwords across devices</string>
     <!-- Preference to access list of saved logins -->
     <string name="preferences_passwords_saved_logins">Saved logins</string>
+    <!-- Preference to access list of saved logins -->
+    <string name="preferences_passwords_saved_logins_2" tools:ignore="UnusedResources">Saved passwords</string>
     <!-- Description of empty list of saved passwords. Placeholder is replaced with app name.  -->
     <string name="preferences_passwords_saved_logins_description_empty_text">The logins you save or sync to %s will show up here.</string>
+    <!-- Description of empty list of saved passwords. Placeholder is replaced with app name.  -->
+    <string name="preferences_passwords_saved_logins_description_empty_text_2" tools:ignore="UnusedResources">The passwords you save or sync to %s will be listed here. All passwords you save are encrypted.
+</string>
     <!-- Preference to access list of saved logins -->
     <string name="preferences_passwords_saved_logins_description_empty_learn_more_link">Learn more about Sync.</string>
+    <!-- Preference to access list of saved logins -->
```

Comment:
Comment does not address string below (it's about sync, not accessing list of saved logins)